src/components/charts/Charts.css

```diff
@@ -0,0 +1,32 @@
+.container {
+    -moz-box-shadow:    0 0 5px 0 #000000;
+  -webkit-box-shadow: 0 0 5px 0 #000000;
+  box-shadow:         0px 0px 2px 2px rgba(0, 0, 0, 0.25);
+  border-radius: 5px;    
+}
+
+.charts {
+  height: 50%;
+  margin-top: 6vh;
+}
+
+.charts-page {
+  display: flex;
+  justify-content: center;
+  overflow-y: auto;
+  height: 93vh;
+}
+
+.large-dropdown {
+  font-size: 18px;
+  border: none;
+  background-color: lightgrey; 
+}
+
+.charts-title {
+  font-size: 18px;
+}
+
+.custom-tooltip {

```

Comment:
no style definition?

Reply:
didn't put any work into styling it but as soon as we do all we need to do is fill that class, it's already connected